core/templates/dev/head/pages/exploration_editor/editor_tab/StateEditorSpec.js

```diff
@@ -145,7 +145,8 @@ describe('State Editor controller', function() {
       ecs.setActiveStateName('First State');
       scope.initStateEditor();
       expect(scope.contentEditorIsOpen).toBe(false);
-      expect(scope.content).toEqual(scope.getContent('First State Content'));
+      expect(angular.equals(
```

Comment:
It seems that it evaluates to false since one is an object whereas the other is a dict